about.html

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <!-- Global site tag (gtag.js) - Google Analytics
+    <script
+      async
+      src="https://www.googletagmanager.com/gtag/js?id=G-YJ2WSX508J"
+    ></script>
+    <script>
+      window.dataLayer = window.dataLayer || [];
+      function gtag() {
+        dataLayer.push(arguments);
+      }
+      gtag("js", new Date());
+      gtag("config", "G-YJ2WSX508J");
+    </script> -->
```

Comment:
- Kindly remove the unused commented lines of code to make your codebase look cleaner.